frontend/__tests__/components/catalog.spec.tsx

```diff
@@ -26,7 +25,7 @@ import {
 import { developerCatalogItems, groupedByType } from './catalog-data';
 import { categorizeItems } from '../../public/components/utils/tile-view-page';
 
-describe(CatalogTileViewPage.displayName, () => {
+describe('Catalog Tile View Page', () => {
```

Comment:
Think this needs a revert @abhinandan13jan

Reply:
yes, reverted